arxiv-auth/pyproject.toml

```diff
@@ -14,7 +14,12 @@ python-dateutil = "*"
 pyjwt = "*"
 redis = "==2.10.6"
 redis-py-cluster = "==1.3.6"
-pydantic = "*"
+pydantic = "^1.0"
```

Comment:
Can we switch to pydantic 2 in new code, if somehow possible?

Reply:
Indeed! Thanks

Reply:
I'd skip upgrading to pydantic v2 to avoid extra work. This code is end of life.